### PR TITLE
Test data necessary for python samples added to test package.

### PR DESCRIPTION
**Pull request by terfendail**
### What does this PR change?
Test launch script updated to run whole set of existing python binding tests.
Samples' data partially added to opencv-test package in addition to extra test data to provide input necessary for new python tests.

### CODE REVIEW

File: cmake/templates/opencv_run_all_tests_unix.sh.in [hunk 1]
```diff
@@ -107,7 +107,7 @@ for t in $OPENCV_PYTHON_TESTS;
 do
     test_name=`basename "$t"`
 
-    cmd="py.test --junitxml $test_name.xml \"$OPENCV_TEST_PATH\"/$t"
+    cmd="python \"$OPENCV_TEST_PATH\"/$t"
```
Comment (asmorkalov) [inline cmake/templates/opencv_run_all_tests_unix.sh.in]: Does it generate some xml file or other testing report?

File: data/CMakeLists.txt [hunk 2]
```diff
@@ -18,5 +18,37 @@ if(INSTALL_TESTS AND OPENCV_TEST_DATA_PATH)
             DIRECTORY_PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE
               GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
             DESTINATION share/OpenCV/testdata COMPONENT tests)
+    if(BUILD_opencv_python)
```
Comment (asmorkalov) [inline data/CMakeLists.txt]: You should also check `INSTALL_TESTS` flag status. If it's disabled there is no reason to install additional test data.

Comment (terfendail) [inline data/CMakeLists.txt]: INSTALL_TESTS is already checked in line 12. Here I've add data necessary for python tests to already packaged test data

File: cmake/templates/opencv_run_all_tests_unix.sh.in (hunk 1 above)
Comment (terfendail) [inline cmake/templates/opencv_run_all_tests_unix.sh.in]: No, it doesn't.

File: data/CMakeLists.txt (hunk 2 above)
Comment (asmorkalov) [inline data/CMakeLists.txt]: I see, never mind.

File: cmake/templates/opencv_run_all_tests_unix.sh.in [hunk 3]
```diff
@@ -107,7 +107,7 @@ for t in $OPENCV_PYTHON_TESTS;
 do
     test_name=`basename "$t"`
 
-    cmd="py.test --junitxml $test_name.xml \"$OPENCV_TEST_PATH\"/$t"
+    cmd="python \"$OPENCV_TEST_PATH\"/$t -v"
```
Comment (asmorkalov) [inline cmake/templates/opencv_run_all_tests_unix.sh.in]: Please remove py.test from deb package dependency though

Comment (terfendail) [inline cmake/templates/opencv_run_all_tests_unix.sh.in]: fixed

**Comment by asmorkalov** [comment]
:+1:

**Pull request merged**